package-lock.json

```diff
@@ -39,7 +39,7 @@
         "eslint-plugin-promise": "^5.1.0",
         "eslint-plugin-react": "^7.21.5",
         "eslint-plugin-react-hooks": "^4.2.0",
-        "flow-bin": "^0.158.0",
```

Comment:
I am unsure of the changes in this file.  Were they really necessary for this PR?  I'm concerned that we're doing too much in the PR if we're both upgrading a library and fixing a bug.

Reply:
Reverted, thanks.